test/common_utils.py

```diff
@@ -87,6 +89,33 @@ def set_audio_backend(backend):
     torchaudio.set_audio_backend(be)
 
 
+class TempDirMixin:
```

Comment:
To avoid inheritance of `PytorchTestCase` multiple times at the test module.
`TempDirMixin` is designed to be composable so that each test case can choose use or not to use.
This decision should not collide with the fact we are requiring each test case to inherit `PytorchTestCase`.